packages/twenty-front/src/modules/object-record/record-table/states/tableCursorByRecordComponentStateV2.ts

```diff
@@ -0,0 +1,11 @@
+import { createComponentFamilyStateV2 } from '@/ui/utilities/state/component-state/utils/createComponentFamilyStateV2';
+import { ViewScopeInternalContext } from '@/views/scopes/scope-internal-context/ViewScopeInternalContext';
+
+// TODO: This is an example that can't be used because the context is not the same between RecordTable and RecordShowPage
+// We should remove it when we have other examples of component family state V2
+export const tableCursorByRecordIdComponentFamilyStateV2 =
```

Comment:
that's why I think we should put all this logic into a "navigation-context" module which declares global state if needed, this is not specific to table